src/content/recipes/@sveltejs/kit.md

```diff
@@ -1,18 +1,12 @@
-<div class="aside aside__no-top">
+<Callout variant="neutral" icon="ℹ️" title="Prerequisites" style={{ marginBottom: '10px' }}>
```

Comment:
Instead of an inline style, can we refactor this CSS to be a better fit for what we're after? Looks like it should target `p`s that immediately following a heading?

```css
p:first-of-type:not(:only-of-type) {
  margin-top: 0;
}
```